### DISCUSSION TITLE
issue: Tab Index is not working when i use update function in UsefieldArray

### DISCUSSION THREAD

**Discussion by NarenderSiri** [react-hook-form/react-hook-form#7940] [Q&A]
### Version Number

7.27.1

### Codesandbox/Expo snack

https://codesandbox.io/s/reacthookformsforsetfocus123-forked-lfjt2p?file=/src/App.js

### Steps to reproduce

1. Go to ' updateData Funtion in App.js'
2. Click on 'tab in the uomCode  text field'
3. See error 'tab index is not working'
4. when comment this update(rowIndex, { ...value });  my code is working

### Expected behaviour

Tab Work in the Usefield

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by NarenderSiri** [comment]
Please sir,
I am trying this from three weeks. 
Please help me

**Reply by bluebill1049** [reply]
Happy to help, but you need to help isolate the problem rather than just dump a half-baked code sandbox and expect us to help on investigation.
